README.md

```diff
@@ -30,6 +30,9 @@ For the complete list of changes made to snowchange check out the [CHANGELOG](CH
 1. [Legal](#legal)
 
 
+## Installation
```

Comment:
Given these changes seems like we need to update a few sections below (under "Running snowchange" and "Integrating With DevOps") with the new pip module. Can you please remove this and update the corresponding sections below?

Reply:
Updated @jeremiahhansen